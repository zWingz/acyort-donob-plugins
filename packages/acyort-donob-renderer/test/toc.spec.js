const fs = require('fs-extra')
const { join } = require('path')
const toc = require('../lib/toc')

describe('test toc', () => {
  it('render toc', () => {
    const content = fs.readFileSync(join(__dirname, './fixtures/parse.md'), {
      encoding: 'utf8',
    })
    const html = toc(content)
    expect(html).toMatchSnapshot('toc')
  })
})
