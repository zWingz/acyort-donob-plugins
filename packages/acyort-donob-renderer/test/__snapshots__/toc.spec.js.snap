// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test toc render toc: toc 1`] = `
"<ul>
<li><a href=\\"#marked-in-the-browsernnrendered-by-marked\\">Marked in the browser\\\\n\\\\nRendered by <strong>marked</strong>.</a><ul>
<li><a href=\\"#section-one\\">Section One</a></li>
<li><a href=\\"#hello\\">hello</a><ul>
<li><a href=\\"#test\\"><code>test</code></a></li>
</ul>
</li>
<li><a href=\\"#%E5%95%8A%E5%95%8A%E5%95%8A\\">啊啊啊</a></li>
</ul>
</li>
</ul>
"
`;
